Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -432,10 +408,6 @@ - (CGSize)cellSizeForViewWidth:(int)viewWidth contentWidth:(int)contentWidth
         cellSize.height += self.footerHeight;
     }
 
-    if (self.shouldHaveFailedSendBadge) {
```

Comment:
Just making sure - this is intentionally going away with no replacement for now?

Reply:
It's on my list of questions to review with Myles.  I'm trying to batch my questions to him (I already have ~30 from just a day's work).